### PR TITLE
Feature: Added bitrate info for audio files in properties windows

### LINKED ISSUE #5566: Feature: Add bitrate info for audio files in properties windows
I just love this app but the biggest flaw in this app is that it doesn't have the bitrate details for an audio file in the details tab. Instead, there's an encoding bitrate but that's not what I need. The only reason I'm considering to switch back to file explorer is this. Hope you will implement it.

**Requirements**
- Display bitrate info in the details page of the properties window

image

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #5566

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/ViewModels/Properties/FileProperties.cs [hunk 1]
```diff
@@ -168,7 +177,7 @@ public async void GetSystemFileProperties()
 			ViewModel.PropertySections = new ObservableCollection<FilePropertySection>(query);
 			ViewModel.FileProperties = new ObservableCollection<FileProperty>(list.Where(i => i.Value is not null));
 		}
-
+
```
Comment (QuaintMako) [inline src/Files.App/ViewModels/Properties/FileProperties.cs:180]:
```suggestion

```

File: src/Files.App/ViewModels/Properties/FileProperties.cs [hunk 2]
```diff
@@ -158,6 +158,15 @@ public async void GetSystemFileProperties()
 
 			list.Find(x => x.ID == "address").Value = await GetAddressFromCoordinatesAsync((double?)list.Find(x => x.Property == "System.GPS.LatitudeDecimal").Value,
 																						   (double?)list.Find(x => x.Property == "System.GPS.LongitudeDecimal").Value);
+			// Find Encoding Bitrate property and convert it to kbps
+			var encodingBitrate = list.Find(x => x.Property == "System.Audio.EncodingBitrate");
+			if (encodingBitrate is not null)
+			{
+				string[] sizes = { "Bps", "KBps", "MBps", "GBps" };
+				int order = (int)Math.Floor(Math.Log((uint)encodingBitrate.Value, 1024));
+				double readableSpeed = (uint)encodingBitrate.Value / Math.Pow(1024, order);
```
Comment (QuaintMako) [inline src/Files.App/ViewModels/Properties/FileProperties.cs:167]: Use `var` for the types when possible.

**Review by QuaintMako — COMMENTED** [review]
Small formatting, but works as intended for me.

**Review by QuaintMako — DISMISSED** [review]

**Review by d2dyno1 — APPROVED** [review]

**Review by yaira2 — DISMISSED** [review]

File: src/Files.App/ViewModels/Properties/FileProperties.cs [hunk 3]
```diff
@@ -168,7 +177,6 @@ public async void GetSystemFileProperties()
 			ViewModel.PropertySections = new ObservableCollection<FilePropertySection>(query);
 			ViewModel.FileProperties = new ObservableCollection<FileProperty>(list.Where(i => i.Value is not null));
 		}
-
```
Comment (yaira2) [inline src/Files.App/ViewModels/Properties/FileProperties.cs:171]: Can you restore this line?

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**